src/components/Sum.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({ a = 0, b = 0 }) => (
```

Comment:
```suggestion
const Sum = ({ a, b }) => (
```

Якщо в тебе є **isERequired**, то тобі не потрібні дефолтні значення